back/app/policies/baskets_idea_policy.rb

```diff
@@ -29,4 +29,8 @@ def update?
   def destroy?
     BasketPolicy.new(user, record.basket).update?
   end
+
+  def upsert?
+    BasketPolicy.new(user, record.basket).update?
```

Comment:
I'm a bit doubting if we also should automatically create a basket. If at some point create? and update? of the basket policy would become different, this would allow bugs to slip in where residents can vote in a situation where they actually shouldn't be allowed to.

Reply:
I do think it will hugely simplify things from a front-end perspective to do this here, so you don't have to do any waiting for the basket to create first in a different request before posting to this endpoint. Again I'll see what Luuc says when he uses the endpoint.

Reply:
Luuc agrees we should create the basket here too - makes the front end much simpler